iceoryx_posh/source/roudi/process_manager.cpp

```diff
@@ -179,7 +179,7 @@ void ProcessManager::evaluateKillError(const Process& process,
     }
 }
 
-bool ProcessManager::registerProcess(const ProcessName_t& name,
+bool ProcessManager::registerProcess(const RuntimeName_t& name,
```

Comment:
name is ok I think, since it is generic and together with the type sufficient. Bbut a specific name like applicationName when the type is a RuntimeName_t might be misleading. At some point we also may have to define rules for this _t suffix, I am no sure we are consistent here and personally I do not like it for class types (as opposed to build-in integral types, pointer types etc.).

Reply:
You're right. This is definitely not consistent. The class `ProcessManager` has a method `registerProcess` which now takes a `RuntimeName_t`. I've added this to #611 

Regarding the `_t` I agree. After the Almond release, I'd vote a at least two refactoring sprints, where we can address things like that.

Reply:
Do you want to rename the ``ProcessManager`` before the 1.0 release?

Reply:
Nope, this PR will be the only one from #611 before the Almond release.